Week3/homework/step4-bonus.js

```diff
@@ -5,6 +5,6 @@ const values = ['a', 'b', 'c', 'd', 'a', 'e', 'f', 'c'];
 // Add your function here. Try and come up with a good name for this function
 
 // Replace `yourFunction` with the name of the function you just created
-const uniqueValues = yourFunction(values);
+const uniqueValues = values.filter((value, index) => values.indexOf(value) === index);
 
 console.log(uniqueValues);
```

Comment:
✅ Nice use of filter.  You are one of the few to use this. Most were using `for` loops.  
I highly recommend going further in the functional programming direction.  My experience is that the more you use higher-order functions the more uses you find, and the better you get and finding and creating utilities that you can use over and over.